### DISCUSSION TITLE
Noob question, I wan to mass edit the series tag

### DISCUSSION THREAD

**Discussion by Kelpgun** [Difegue/LANraragi#503] [Q&A] [answered]
I ran the Filename Parsing plugin on my database and it gave all the manga that had a magazine name in the filename the series:comic(etc).
How would I go about changing the series tag to a magazine tag en mass?
(I tried doing find and replace with notepad++ on a copy of the database but it broke it)

**Comment by Difegue** [comment, marked as answer]
They haven't landed in a numbered release yet, but if you're willing to use nightly builds, [Tag Rules](https://sugoi.gitbook.io/lanraragi/v/dev/advanced-usage/tag-rules) can exactly cover this usecase. 👍  

Keep in mind tag rules only apply to incoming tags and won't modify what's already there.

**Reply by Kelpgun** [reply]
Thanks for answering! I found a workaround by backing up the database to a json file, applying the changes I wanted to it and restoring from that
